### ISSUE TITLE
Support for INA228 on v5x Power Module. 

### ISSUE THREAD

**Issue by vincentpoont2** [PX4/PX4-Autopilot#17853]
Would like to add INA228 power monitor to Holybro v5x power module for support up to 12s battery.

Can we add support for this on PX4?

**Comment by dagar** [comment]
Yes that shouldn't be a problem. Please let us know when hardware is available for testing and I'll put something together.

**Comment by vincentpoont2** [comment]
Thanks @dagar, will send it over once its ready.
